setup/docker/slurm/cgroup.conf

```diff
@@ -0,0 +1 @@
+CgroupPlugin=cgroup/v1
```

Comment:
You might be able to just `RUN echo "CgroupPlugin=cgroup/v1" >> /etc/slurm/cgroup.conf` in the Dockerfile.

Reply:
I'd like to get away from scripts emitting scripts where possible, so even though this is a pretty simple file, it makes it clear what it's values are without having to hunt around in the Dockerfile